### ISSUE TITLE
add default password for each protocol - particularly windows and putty

### ISSUE THREAD

**Issue by ray-brum** [mRemoteNG/mRemoteNG#2320]
We have personal accounts on windows and linux that are different and changed monthly.
Currently I add the windows account as the default addount/password, but have to override the unix servers with the linux account and change the password each month

I suspect many shops have a similar setup

would it be a good idea to be able to set a default account/password for each protocol (RDP/ PUTTY / etc).

* Version used: 1.76.20
* Operating System: multiple versions of windows on a domain + linux 7.9 on a "domain"
[mremoteng setup.docx](https://github.com/mRemoteNG/mRemoteNG/files/10344523/mremoteng.setup.docx)

**Comment by ray-brum** [comment]
apparently a similar request has been made already (targetted at 1.78.*). Would default password for each protocol be easier to implement? However the other suggestion would resolve my problem too

**Comment by simonai1254** [comment]
I would consider this a duplicate of #2304, even though this is not protocol based. The profile solution planned to be implemented by @Kvarkas should remediate this issue and might allow the extension to define a defeault user for each connection type.

**Comment by Kvarkas** [comment]
yes planed profiles implem,entation to be done, after that you will be able to set a profile of dewfault connection (you may have several) and that profile will "know" where to look for default password and for that user profile

**Comment by robertpopa22** [comment]
**Fix available for testing**

**Commit:** `60f8c5e5` (fork `robertpopa22/mRemoteNG`, branch `main`)
**What changed:** add default password for each protocol - particularly window

A beta build (v1.81.0-beta.4) including this fix is available from the fork's Releases page.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
